app/javascript/crayons/MentionAutocomplete/MentionAutocompleteCombobox.jsx

```diff
@@ -127,6 +136,9 @@ export const MentionAutocompleteCombobox = ({
       <ComboboxInput
         style={{
           opacity: 0.000001,
+          position: 'absolute',
+          top: placementCoords.y,
+          left: placementCoords.x,
```

Comment:
This input isn't visible, but placing it here stops any scroll jumps shows correct outline when a screen reader user is interacting with it

Reply:
Could we embed this note as a comment? I think it would improve legibility as to the why we pass these attributes

Reply:
I've updated the JSDoc comments for the `placementCoords` prop to be a bit more descriptive about how its used